lighthouse-core/audits/autocomplete.js

```diff
@@ -43,22 +194,80 @@ class AutocompleteAudit extends Audit {
     };
   }
 
+  /**
+   * @param {LH.Artifacts.FormInput} input
+   * @return {{hasValidTokens: Boolean, inValidOrder?: Boolean}}
+   */
+  static checkAttributeValidity(input) {
+    if (!input.autocomplete.attribute) return {hasValidTokens: false};
+    const tokenArray = input.autocomplete.attribute.split(' ');
+    for (const token of tokenArray) {
+      // A `section-` prefix indicates a unique autofill scope.
+      // https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#:~:text=section-
+      if (token.slice(0, 8) === 'section-') continue;
+      if (validAutocompleteTokens.includes(token)) continue;
+      return {hasValidTokens: false};
+    }
+    // If all autocomplete tokens are valid but there is still no property attribute, then that means the tokens are out of order.
+    // https://cloudfour.com/thinks/autofill-what-web-devs-should-know-but-dont/#all-the-tokens
+    if (!input.autocomplete.property) return {hasValidTokens: true, inValidOrder: true};
+    return {hasValidTokens: true};
+  }
+
   /**
    * @param {LH.Artifacts} artifacts
    * @return {LH.Audit.Product}
    */
   static audit(artifacts) {
     const forms = artifacts.FormElements;
     const failingFormsData = [];
+    const warnings = [];
+    let notApplicable = 0;
```

Comment:
we can capture this same thing with just a boolean `let foundPrediction = false`. Set to true when found.

Reply:
ohhh smart, on it